composer.json

```diff
@@ -18,8 +18,8 @@
     "behat/behat": "~3.1.0-rc2",
     "behat/mink-extension": "~2.0",
     "drupal/drupal-driver": "dev-master",
-    "symfony/dependency-injection": "~2.7",
-    "symfony/event-dispatcher": "~2.7"
+    "symfony/dependency-injection": "2.8.*",
+    "symfony/event-dispatcher": "2.8.*"
```

Comment:
This would break compatibility with `8.0.*` (and future compatibility when core upgrades symfony). By using the `~`, individual users of this library are able to pin exact versions if needed.